src/recommendpolicy/recommendPolicy.go

```diff
@@ -180,3 +176,43 @@ func uniqueNsDeploy(deployName, deployNamespace string) *types.Deployment {
 
 	return &deploy
 }
+
+func GetHardenPolicy(deployments *v1.DeploymentList, replicaSets *v1.ReplicaSetList, statefulSets *v1.StatefulSetList, nsNotFilter []string) []types.KnoxSystemPolicy {
+
+	var policies []types.KnoxSystemPolicy
+	if !isLatest() {
+		if _, err := DownloadAndUnzipRelease(); err != nil {
+			log.Error().Msgf("Unable to download %v", err.Error())
```

Comment:
Should this continue if the operation fails?

Reply:
It would be good to print the version returned by `DownloadAndUnzipRelease` so as to track.